csv/dup.csv

```diff
@@ -0,0 +1,5 @@
+Name,Total,Pass,Time
+is_sized,20200,10073,0.1s
+is_sorted,20000,20000,0.1s
+is_duplicate,20000,20000,0.1s
```

Comment:
In this case (I think) we see that a generator that produces only duplicate lists also produces sorted lists. 

An interesting and important question is what can we say empirically that something is being missed here? How do we check that the full range of sorted lists is being produced?(Besides saying their coverage types are different)